webpack.config.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path');
-const TerserPlugin = require('terser-webpack-plugin');
 const webpack = require('webpack');
+const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
```

Comment:
Nested deps of `react-markdown` rely on some Node.js functions to be present or polyfilled. Up to Webpack 4 these polyfills were added automatically, we now have to do so explicitly.